### PR TITLE
Updated language use in goto-cc.md

### PR DESCRIPTION
**Pull request by edstenson**
- [X] Each commit message has a non-empty body, explaining why the change was made.
- [X] NOT RELEVANT Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [X] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [X] NOT RELEVANT Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [X] NOT RELEVANT My commit message includes data points confirming performance improvements (if claimed).
- [X] My PR is restricted to a single feature or bugfix.
- [X] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by allredj — COMMENTED** [review]
:no_entry_sign:
This PR failed Diffblue compatibility checks (cbmc commit: fc19ea072cf5470475efd73cdedf180f6dda0ae2).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/93879867
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

The incompatibility may have been introduced by an earlier PR. In that case merging this
PR should be avoided unless it fixes the current incompatibility.

File: doc/cprover-manual/goto-cc.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
 Existing software projects usually do not come in a single source file
-that may simply be passed to a model checker. They rather come in a
+that may simply be passed to a model checker. Rather, they come in a
 multitude of source files in different directories and refer to external
 libraries and system-wide options. A *build system* then collects the
 configuration options from the system and compiles the software
 according to build rules.
 
-The most prevalent build tool on Unix (-based) systems surely is the
+The most prevalent build tool on Unix-based systems ia the
```
Comment (tautschnig) [inline doc/cprover-manual/goto-cc.md]: "is"

**Review by tautschnig — CHANGES_REQUESTED** [review]

**Review by allredj — COMMENTED** [review]
:no_entry_sign:
This PR failed Diffblue compatibility checks (cbmc commit: 992d6c286359aac298f831176358f52d99f0d576).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/93987370
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

The incompatibility may have been introduced by an earlier PR. In that case merging this
PR should be avoided unless it fixes the current incompatibility.

**Review by allredj — COMMENTED** [review]
:no_entry_sign:
This PR failed Diffblue compatibility checks (cbmc commit: 36749085640f630c856589025837688b8f25a3ea).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/93990166
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

The incompatibility may have been introduced by an earlier PR. In that case merging this
PR should be avoided unless it fixes the current incompatibility.

**Review by tautschnig — APPROVED** [review]

**Pull request merged**